akka-actor/src/main/scala/akka/dispatch/Mailbox.scala

```diff
@@ -226,21 +225,6 @@ private[akka] abstract class Mailbox(val messageQueue: MessageQueue)
     }
   }
 
-  override final def getRawResult(): Unit = ()
```

Comment:
is this needed? It was in the reverse patch and it does not seem to affect anything, but some confirmation would be helpful